### DISCUSSION TITLE
Localization of ttkbootstrap

### DISCUSSION THREAD

**Discussion by vmakrov1720** [israel-dryer/ttkbootstrap#510] [Show and tell]
Can anyone help me to implement the localization in french
 with a simple sample? I am a newbie in python.
(C:\Python310\Lib\site-packages\ttkbootstrap\localization\msgs.py)
Thanks a lot!

**Comment by TheDemonGuard** [comment]
@vmakrov1720 

French localization is already implemented with PR #361 

In case your system is configured with French locale, you should see French on the implemented UI component.
Otherwise, you can override the system default locale and force Frech locale, you can do so by setting 'fr' to "MessageCatalog.locale" function call.
ex: MessageCatalog.locale('fr')

Sample script as follows:

        import ttkbootstrap as ttkb
        from ttkbootstrap.dialogs import DatePickerDialog
        from datetime import date
        from ttkbootstrap.localization import MessageCatalog

        root = ttkb.Window(themename="flatly", minsize=(500, 500))

        # System default locale
        date_picker_dialog = DatePickerDialog(parent=None, title='Date Picker Dialog', firstweekday=6, startdate=date.today(), bootstyle='info')

        # Override system default locale with French
        MessageCatalog.locale('fr')
        date_picker_dialog = DatePickerDialog(parent=None, title='Date Picker Dialog', firstweekday=6, startdate=date.today(), bootstyle='info')

        # Override system default locale with Japanese
        MessageCatalog.locale('ja')
        date_picker_dialog = DatePickerDialog(parent=None, title='Date Picker Dialog', firstweekday=6, startdate=date.today(), bootstyle='info')

        root.mainloop()

> Can anyone help me to implement the localization in french with a simple sample? I am a newbie in python. (C:\Python310\Lib\site-packages\ttkbootstrap\localization\msgs.py) Thanks a lot!
